### PR TITLE
Amazon aws images

### PR DESCRIPTION
**Pull request by mihaitusa**

### CODE REVIEW

File: score-jClouds/src/main/java/io/cloudslang/content/jclouds/services/impl/imagesImpl/AmazonImageService.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package io.cloudslang.content.jclouds.services.impl.imagesImpl;
```
Comment (gituser173) [inline score-jClouds/src/main/java/io/cloudslang/content/jclouds/services/impl/imagesImpl/AmazonImageService.java]: why do you need imagesImpl package? why not keep them in the same io.cloudslang.content.jclouds.services.impl package?

File: score-jClouds/src/main/java/io/cloudslang/content/jclouds/services/impl/ComputeServiceImpl.java [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
 import java.util.HashSet;
 import java.util.Set;
 
 /**
  * Created by persdana on 6/5/2015.
  */
-public class ComputeServiceImpl extends JcloudsComputeService implements ComputeService{
-    private static final String NOT_IMPLEMENTED_ERROR_MESSAGE = "Not implemented. Use 'amazon\' or 'openstack' providers in the provider input";
-
+public class ComputeServiceImpl extends JCloudsComputeService implements ComputeService {
```
Comment (gituser173) [inline score-jClouds/src/main/java/io/cloudslang/content/jclouds/services/impl/ComputeServiceImpl.java]: maybe you could rename all the service implementations using the "Impl" suffix like you did here? WDYT?

Comment (mihaitusa) [inline score-jClouds/src/main/java/io/cloudslang/content/jclouds/services/impl/ComputeServiceImpl.java]: @gituser173 Same as above.

File: score-jClouds/src/main/java/io/cloudslang/content/jclouds/services/impl/imagesImpl/AmazonImageService.java (hunk 1 above)
Comment (mihaitusa) [inline score-jClouds/src/main/java/io/cloudslang/content/jclouds/services/impl/imagesImpl/AmazonImageService.java]: @gituser173  Will discuss this offline.

**Comment by gituser173** [comment]
Looks good, but I advise that in the future you do smaller check-ins, as it is easy to overlook mistakes when reviewing 92 files

**Pull request merged**